inappmessaging/src/main/java/com/rakuten/tech/mobile/inappmessaging/runtime/data/models/appevents/Event.kt

```diff
@@ -50,4 +50,14 @@ interface Event {
     @RestrictTo(LIBRARY)
     @NotNull
     fun getAttributeMap(): Map<@NotNull String, @Nullable Attribute?>
+
+    /**
+     * This method returns true if the event was logged when user information was updated.
+     */
+    fun isUserUpdated(): Boolean
```

Comment:
To be honest I don't like the fact that Event becomes aware of the existence of User  (reference without a clear relation between models). I know that's the easiest solution but maybe it's better for one of event repositories to store a tuple/dict of event-flag pairs instead. (Unless this is a temporary solution before refactoring. If that's the case then it's ok)

Reply:
Yeah, I don't like this too. But yes, this is a temporary solution which will be updated after refactoring event and campaign repositories to sync with iOS.